core/src/main/java/org/apache/james/core/healthcheck/ResultStatus.java

```diff
@@ -18,6 +18,28 @@
  ****************************************************************/
 package org.apache.james.core.healthcheck;
 
+import java.util.Arrays;
+
 public enum ResultStatus {
-    HEALTHY, DEGRADED, UNHEALTHY;
+    HEALTHY("healthy"), 
+    DEGRADED("degraded"), 
+    UNHEALTHY("unhealthy");
+    
+    public static ResultStatus fromString(String value) {
```

Comment:
I don't see the place you use that method. Could you verify it? If none, then we could remove this method

Reply:
fixed in 496dd03a22c1cd7c6a45069bed55b511511bf09c